doc/interfaces/registration_flow.rst

```diff
@@ -0,0 +1,163 @@
+we would call the ``dipy_align_affine`` command as, e.g.::
+
+    dipy_align_affine <path_to_static_file> <path_to_moving_file> --transform "affine" --out_dir "affine_reg_output" --out_affine "affine_reg.txt"
+
+This command will apply an affine transformation on the moving image file, and
+save the transformed image and the affine matrix to the ``affine_reg_output``
+directory.
+
+In case you did not specify the output directory, the transformed image file
+and affine matrix would be saved to the current by default. If you did not
+specify the name of the output affine matrix, the affine matrix will be saved
+to a file named ``affine.txt`` by default, located in the current directory
+also by default.
+
+------------------------------------
+Symmetric Diffeomorphic Registration
+------------------------------------
+
+Symmetric Diffeomorphic Registration is performed using the Symmetric
+Normalization (SyN) algorithm proposed by Avants et al. [Avants09]_ (also
+implemented in the ANTs software [Avants11]_). It is an optimization technique
+that brings the moving image closer to the static image.
+
+Create a directory in which to save the transformed image (e.g.:
+``syn_reg_output``)::
+    
+    mkdir syn_reg_output
+
+To run the symmetric normalization registration method, we need to specify the
+paths to the static image file, and to the moving image file, followed by
+optional arguments. In this case, we will be specifying the metric (``metric``),
+the output directory (``out_dir``) and the file name of the output warped image
+(``out_warped``). You can use cc (cross correlation), ssd (sum squared
+differences) or em (expectation-maximization) as metrics.
+
+The symmetric diffeomorphic registration method in DIPY is run through the
+``dipy_align_syn`` command, e.g.::
+
+    dipy_align_syn <path_to_static_file> <path_to_moving_file> --metric "cc" --out_dir "syn_reg_output" --out_warped "syn_reg_warped.nii.gz"
+
+In case you did not specify the output directory, the transformed files would
+be saved in the current directory by default. If you did not specify the file
+name of the output warped image, the warped file will be saved as
+``warped_moved.nii.gz`` by default.
+
+----------------------
+Apply a Transformation
+----------------------
+
+We can apply a transformation computed previously to an image. In order to do
+so, we need to specify the path of the static image file, moving image file,
+and transform map file followed by optional arguments. In this case, we will
+be specifying the transform type (``transform_type``) and the output directory
+(``out_dir``).
+
+Create a directory in which to save the transfomed files (e.g.:
+``transform_reg_output``)::
+
+    mkdir transform_reg_output
+
+For a ``diffeomorphic`` transformation, we would run the command as::
+
+    dipy_apply_transform <path_to_static_file> <path_to_moving_file> <path_to_transform_map_file> --transform_type "diffeomorphic" --out_dir "transform_reg_output"
```

Comment:
Missing a small information to tell what is the `transform_map_file`

Reply:
Addressed in 747dd55